### DISCUSSION TITLE
Flutter version of the library 💙

### DISCUSSION THREAD

**Discussion by moazelsawaf** [dofire/Floating-Bubble-View#20] [Show and tell]
![package_banner](https://user-images.githubusercontent.com/43591891/222908927-83beeb18-ac8a-4ad3-bf2e-bd7cba941470.png)

### Hello Everyone 👋🏻💙

I am glad to share with you a Flutter version from the library under the name of [dash_bubble](https://github.com/moazelsawaf/dash_bubble), the plugin depends on this library in its implementation 💪🏻💙

I tried to do my best developing the plugin to ensure the highest quality, code documentation, and ease of use 💙

I have also referenced this library and author in the first lines of the README of the plugin and would be happy if my plugin gets referenced back in the README of this library as a Flutter Version of the library as done in this [blade_flags](https://github.com/MohmmedAshraf/blade-flags#flutter-version) Laravel Package 💙

Here is the link of the plugin in pub.dev (The official package repository for [Dart](https://dart.dev/) and [Flutter](https://flutter.dev/) apps):

https://pub.dev/packages/dash_bubble

I hope you like it ... 💙

Thanks 🙏🏻💙
Moaz El-sawaf

**Comment by dofire** [comment]
Hi Moaz El-sawaf,

Thanks for sharing the Flutter version (dash_bubble) of this library! I really appreciate the time and effort you've put into developing the plugin to ensure that it has good documentation and is easy to use.

I'm also glad to hear that you've credited this library and author in the README of the plugin. I'd be more than happy to link your plugin as a Flutter Version of this library, as I believe in supporting one another's work.

Congratulations on the successful launch of dash_bubble on pub.dev! I'm sure that it will be a valuable resource for the Flutter community.

Thanks again for choosing this library. I'm excited to see what else you'll create! ♥

**Reply by moazelsawaf** [reply]
Thank you so much for your kind words and I hope we achieve a lot of success together 🙏🏻❤️
